tools/project_manager/editproject.php

```diff
@@ -1016,12 +1016,12 @@ function preview()
         echo "<h2 id='preview'>", _("Preview Project"), "</h2>";
         echo "<p>", _("This is a preview of your project and roughly how it will look to the proofreaders."), "</p>\n";
         echo "<table class='basic'>";
-        echo "<tr><th>", _("Title"), "</th><td>$this->nameofwork</td></tr>\n";
-        echo "<tr><th>", _("Author"), "</th><td>$this->authorsname</td></tr>\n";
+        echo "<tr><th>", _("Title"), "</th><td>", html_safe($this->nameofwork), "</td></tr>\n";
+        echo "<tr><th>", _("Author"), "</th><td>", html_safe($this->authorsname), "</td></tr>\n";
         if (user_is_a_sitemanager())
         {
             // SAs are the only ones who can change this.
-            echo "<tr><th>", _("Project Manager"), "</th><td>$this->projectmanager</td></tr>\n";
+            echo "<tr><th>", _("Project Manager"), "</th><td>", html_safe($this->projectmanager), "</td></tr>\n";
```

Comment:
You shouldn't need to escape this, these must be valid usernames which can't contain HTML.